contracts/examples/PullPaymentCapableExample.sol

```diff
@@ -0,0 +1,9 @@
+import '../PullPaymentCapable.sol';
+
+// Example class using PullPaymentCapable
+contract PullPaymentCapableExample is PullPaymentCapable {
```

Comment:
Not necessarily. This was required for this particular test, because `PullPaymentCapable` has internal functions that can only be called from a contract inheriting from it.